robotbase/src/main/java/frc/robot/subsystems/AlgaePivot.java

```diff
@@ -9,18 +9,21 @@
 import com.revrobotics.spark.SparkMax;
 import edu.wpi.first.units.Units;
 import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.units.measure.MutAngle;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.ALGAE_PIVOT;
 import frc.robot.Constants.CAN_ID;
 
 public class AlgaePivot extends SubsystemBase {
 
-  private Angle m_targetAngle;
   private SparkMax m_leftMotor;
   private SparkMax m_rightMotor;
   private SparkAbsoluteEncoder m_absoluteEncoder;
   private SparkClosedLoopController m_leftPidController;
 
+  private MutAngle m_targetAngle = Units.Degrees.mutable(Double.NaN);
```

Comment:
Update variable name to m_targetAngleHolder to match the other names we have for mutable units

Reply:
The reason I did not change this was that this variable is used for logic in the subsystem. The holder variables are just there to avoid excess instantiation.